packages/console/src/types/theme.ts

```diff
@@ -0,0 +1,4 @@
+export enum Theme {
+  LightMode = 'light',
```

Comment:
Should we assign the `AppearanceMode.LightMode` to the Theme directly?
?

Reply:
Explicit enum value must only be a literal value (string, number, boolean, etc), cannot be something like `AppearanceMode.LightMode`

Reply:
Union type? Better keep an SSOT of the light/dark string definition.

Reply:
After offline discussions, it also makes sense to separate the definitions of the two.